### PR TITLE
Move index to src

### PR DESCRIPTION
**Pull request by alfetopito**
Follow up to https://github.com/cowprotocol/robot-cow-bot/pull/2#issuecomment-1129854154

Moving `index.js` to `src/index.js`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Out of the scope of this README, but in summary [something like this](https://ww
 
 ```bash
 docker build . -t robot-cow-bot
-docker run -d robot-cow-bot
+docker run -d --env-file .env robot-cow-bot
```
Comment (anxolin) [inline README.md:40]: in this pr u haven't added the instructions, or dotenv library. I like this change, but is this part of another PR?

File: package.json [hunk 2]
```diff
@@ -17,6 +17,6 @@
   },
   "scripts": {
     "test": "echo 'fuck off' && exit",
```
Comment (anxolin) [inline package.json]: are we sure we want to write bad sounding words :P

File: README.md (hunk 1 above)
Comment (alfetopito) [reply README.md:40]: Well, now with the move to a different folder for some reason docker refuses to pick up the env file.
Had to explicitly set it as a `docker run` parameter to get it working, thus, added here.

File: package.json (hunk 2 above)
Comment (alfetopito) [reply package.json]: Ok ok... will make it nicer

**Pull request merged**